CMake/External_Caffe.cmake

```diff
@@ -240,6 +242,37 @@ else()
 endif()
 
 # Main build and install command
+if(WIN32)
+#link_libraries(${fletch_BUILD_PREFIX}/src/Caffe-build/libraries/lib)
+
+ExternalProject_Add(Caffe
+  DEPENDS ${Caffe_DEPENDS}
+  URL ${Caffe_url}
+  URL_MD5 ${Caffe_md5}
+  PREFIX ${fletch_BUILD_PREFIX}
+  DOWNLOAD_DIR ${fletch_DOWNLOAD_DIR}
+  INSTALL_DIR ${fletch_BUILD_INSTALL_PREFIX}
+
+  PATCH_COMMAND ${CMAKE_COMMAND}
+    -DCaffe_patch=${fletch_SOURCE_DIR}/Patches/Caffe
+    -DCaffe_source=${fletch_BUILD_PREFIX}/src/Caffe
+    -P ${fletch_SOURCE_DIR}/Patches/Caffe/Patch.cmake
+
+  CMAKE_COMMAND
+  CMAKE_GENERATOR ${gen}
+  CMAKE_ARGS
+    ${COMMON_CMAKE_ARGS}
+    -DCMAKE_CXX_COMPILER:PATH=${CMAKE_CXX_COMPILER}
+    -DCMAKE_C_COMPILER:PATH=${CMAKE_C_COMPILER}
+    -DBOOST_ROOT:PATH=${BOOST_ROOT}
+	-DBoost_USE_STATIC_LIBS:BOOL=OFF
+    -DBLAS:STRING=Open
+	-DBUILD_SHARED_LIBS:BOOL=ON
```

Comment:
Blegh, notepad++ with default settings is terrible about messing up whitespaces. Easy fix at least.